src/config/utils.ts

```diff
@@ -1,51 +1,42 @@
 import * as _ from 'lodash';
+import * as Bluebird from 'bluebird';
 
+import * as config from '../config';
 import * as constants from '../lib/constants';
 import { getMetaOSRelease } from '../lib/os-release';
 import { EnvVarObject } from '../lib/types';
-import { ExtlinuxConfigBackend } from './backends/extlinux';
-import { ExtraUEnvConfigBackend } from './backends/extra-uEnv';
-import { RPiConfigBackend } from './backends/raspberry-pi';
-import { ConfigfsConfigBackend } from './backends/config-fs';
-import { ConfigOptions, DeviceConfigBackend } from './backends/backend';
+import { Extlinux } from './backends/extlinux';
+import { ExtraUEnv } from './backends/extra-uEnv';
+import { ConfigTxt } from './backends/config-txt';
+import { ConfigFs } from './backends/config-fs';
```

Comment:
Probably it's worth adding an `index.ts` at this point which exports all of these.